Source/UI/NewUI/CommonUI/Protocols/Elements/ElementButtonStyle.swift

```diff
@@ -10,6 +10,8 @@ public protocol ElementButtonStyle: ElementStyle {
     var focusBorderColor: UIColor { get }
     var errorBorderColor: UIColor { get }
     var image: UIImage? { get }
+    var cornerRadius: CGFloat { get }
```

Comment:
do we want to update unit test to add these properties ?

Reply:
Added